Tests/CMakeOnly/GNUInstallDirs2Other/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.2.3)
+
+project(GNUInstallDirsTest)
```

Comment:
Unfortunately it'll cause an error like this:

    CMake Warning (dev) at /usr/share/cmake/Modules/GNUInstallDirs.cmake:177 (message):
      Unable to determine default CMAKE_INSTALL_LIBDIR directory because no
      target architecture is known.  Please enable at least one language before
      including GNUInstallDirs.